backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -281,4 +281,42 @@ public UserInfo getCurrentUserInfo() {
             })
         .collect(Collectors.toList());
   }
+
+  @AuthorizationConfiguration.RequirePermissionManageUsers
```

Comment:
this should be `RequirePermissionManageTargetUser`, and then change the parameter name to `userId`

Reply:
This would allow site admins to make this call, though it will likely end up failing with a `MisconfiguredUserException`.  That seems okay to me, but is a little weird since it means that site admins will be denied `getUsersInCurrentOrg()`, but can access more detailed data with `getUserInCurrentOrg(userId)`.

Should a new authorization verifier be added that is similar to `RequirePermissionManageTargetUser` but does not allow site admins?

Reply:
a) i don't think it would yield a `MisconfiguredUserException`, based on where I see that used in the code today. If you look at the definition of the `RequirePermissionManageTargetUser`, it would hit the site admin evaluation and succeed
b) i would rather change the existing permission annotations to allow both `RequirePermissionManageTargetUser` and `RequirePermissionManageUsers` to succeed for site admins

Reply:
The problem really stems from assuming that we're operating on the current organization.  The admin user may be in no organization (giving `MisconfiguredUserException`) or could be in a different organization (causing incorrect response).

I think in the case of a site admin making the request we'd need to go to okta to find the target user's organization and build the response from that.